server/models/article.js

```diff
@@ -32,11 +36,16 @@ module.exports = (sequelize, DataTypes) => {
       defaultValue: []
     },
     userId: {
-      type: DataTypes.UUID
+      type: DataTypes.UUID,
+      references: {
+        model: 'User',
+        key: 'id',
+      },
     },
   }, {});
   Article.associate = (models) => {
     Article.belongsTo(models.User, {
+      as: 'user',
       foreignkey: 'userId',
```

Comment:
There also seems to be some inconsistency in 'foreignkey' and 'foreignKey' use. I think we should stick to 'foreignKey'. It follows convention.

Reply:
> There also seems to be some inconsistency in 'foreignkey' and 'foreignKey' use. I think we should stick to 'foreignKey'. It follows convention.

I second this. I think we should post it on the team channel.

Reply:
> > There also seems to be some inconsistency in 'foreignkey' and 'foreignKey' use. I think we should stick to 'foreignKey'. It follows convention.
> 
> I second this. I think we should post it on the team channel.

Resolved